### PR TITLE
Alchemistry

### PR DESCRIPTION
**Pull request by CrankySupertoon**

### CODE REVIEW

File: docs/Mods/Alchemistry/ChemicalCombiner.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Chemical Combiner
+
+# Package
+`mods.alchemistry.Combiner`
+
+## Methods
+- **[IItemStack](/Vanilla/Items/IItemStack/) output** - The output of the recipe.
+
+- **[IItemStack](/Vanilla/Items/IItemStack/) input** - The input of the recipe.
```
Comment (kindlich) [inline docs/Mods/Alchemistry/ChemicalCombiner.md]: Inconsistent Types:
Here you write IItemStack, below you use IItemStack[]

File: docs/Mods/Alchemistry/Dissolver.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Dissolver
+
+## Package
+`mods.alchemistry.Dissolver`
+
+## Methods
+- **[IItemStack](/Vanilla/Items/IItemStack/) input** - The input of the recipe.
+- **boolean relativeProbability** - Possibility Enable / Disable
+- **int rolls** -  Possibilty of Roles
+
```
Comment (kindlich) [inline docs/Mods/Alchemistry/Dissolver.md]: Inconsistent Style:
For most of other methods you specify every parameter, here you omit `probabilityGroups`

File: docs/Mods/Alchemistry/Electrolyzer.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+# Package
+`mods.alchemistry.Electrolyzer`
+
+## Methods
+- **[ILiquidStack](/Vanilla/Liquids/ILiquidStack/) input** - The liquid input of the recipe.
+- **[IItemStack](/Vanilla/Items/IItemStack/) electrolyte** - The electrolyte input of the recipe.
+- **int electrolyteConsumptionChance** - Chance of Electrolye being consumed
+- **[IItemStack](/Vanilla/Items/IItemStack/) output** - Output for the recipe
+
```
Comment (kindlich) [inline docs/Mods/Alchemistry/Electrolyzer.md]: Did you omit output and chance 2~4 on purpose?

Is the chance in percent, or is it weight-based?

File: docs/Mods/Alchemistry/Liquifier.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Liquifier
+
+# Package
+`mods.alchemistry.Liquifier`
+
+## Methods
+**[IItemStack](/Vanilla/Items/IItemStack/) output** - The output of the recipe.
```
[the remark this answers is not in the document]
Comment (CrankySupertoon) [reply docs/Mods/Alchemistry/Liquifier.md]: 👍

File: docs/Mods/Alchemistry/Electrolyzer.md (hunk 3 above)
Comment (CrankySupertoon) [reply docs/Mods/Alchemistry/Electrolyzer.md]: No clue I skimmed the code, I don’t know how it works @al132mc

File: docs/Mods/Alchemistry/Dissolver.md (hunk 2 above)
Comment (CrankySupertoon) [reply docs/Mods/Alchemistry/Dissolver.md]: Sorry I’m confused I’m referring to his file for this machine https://github.com/al132mc/alchemistry/blob/1.12.2/src/main/java/al132/alchemistry/compat/ct/CTChemicalDissolver.kt

File: docs/Mods/Alchemistry/Electrolyzer.md (hunk 3 above)
Comment (CrankySupertoon) [reply docs/Mods/Alchemistry/Electrolyzer.md]: Output 3 and 4 are optional

File: docs/Mods/Alchemistry/Dissolver.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# Dissolver
+
+## Package
+`mods.alchemistry.Dissolver`
+
+## Methods
+- **[IIngredient](/Vanilla/Variable_Input/IIngredient/) input** - The input of the recipe.
```
Comment (kindlich) [inline docs/Mods/Alchemistry/Dissolver.md]:
```suggestion
- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) input** - The input of the recipe.
```

That's what travis is for, catching incorrect links ^^

**Comment by kindlich** [comment]
Btw, if you are on discord I can help you with setting up git so you can create larger commits at once (I think you currently only use the GH web user interfaces?)

**Comment by CrankySupertoon** [comment]
Nah I have git  installed. I’m been doing most smaller stuff on mobile and the first commit was done on my desktop client via my GPG Key

**Comment by CrankySupertoon** [comment]
I’ll close this and squash it all tomorrow

**Comment by CrankySupertoon** [comment]
Well close it after I finish

**Comment by CrankySupertoon** [comment]
#275

**Pull request closed without merging**